lib/completions/endpoints/base.rb

```diff
@@ -16,6 +16,7 @@ def self.endpoint_for(model_name)
             DiscourseAi::Completions::Endpoints::OpenAi,
             DiscourseAi::Completions::Endpoints::HuggingFace,
             DiscourseAi::Completions::Endpoints::Gemini,
+            DiscourseAi::Completions::Endpoints::Vllm,
```

Comment:
Remember that we still rely on order here (will change in the future as we discussed). As it is right now, we'll always use HF for these models.

Reply:
Yup that's the plan. Customers may use HF so it takes priority.